### ISSUE TITLE
[bug] Example app is crashing in Android emulator

### ISSUE THREAD

**Issue by wildsurfer** [Firelands128/photo_gallery#2]
I'm trying to run example app in Android emulator. App is crashing when I click on album thumbnail. Unfortunately I don't have any bare metal android device to check if it crashes as well. 

```
Launching lib/main.dart on sdk gphone x86 in debug mode...
Running Gradle task 'assembleDebug'...
✓ Built build/app/outputs/apk/debug/app-debug.apk.
Debug service listening on ws://127.0.0.1:63078/ynrTcu6F2FE=/ws
Syncing files to device sdk gphone x86...
E/AndroidRuntime( 7800): FATAL EXCEPTION: AsyncTask #1
E/AndroidRuntime( 7800): Process: com.morbit.photo_gallery_example, PID: 7800
E/AndroidRuntime( 7800): java.lang.RuntimeException: An error occurred while executing doInBackground()
E/AndroidRuntime( 7800): 	at android.os.AsyncTask$4.done(AsyncTask.java:415)
E/AndroidRuntime( 7800): 	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
E/AndroidRuntime( 7800): 	at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
E/AndroidRuntime( 7800): 	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
E/AndroidRuntime( 7800): 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
E/AndroidRuntime( 7800): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/AndroidRuntime( 7800): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/AndroidRuntime( 7800): 	at java.lang.Thread.run(Thread.java:923)
E/AndroidRuntime( 7800): Caused by: java.io.FileNotFoundException: /storage/emulated/0/Download/IMG_0182 (1).jpg: open failed: ENOENT (No such file or directory)
E/AndroidRuntime( 7800): 	at android.database.DatabaseUtils.readExceptionWithFileNotFoundExceptionFromParcel(DatabaseUtils.java:151)
E/AndroidRuntime( 7800): 	at android.content.ContentProviderProxy.openTypedAssetFile(ContentProviderNative.java:781)
E/AndroidRuntime( 7800): 	at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:1983)
E/AndroidRuntime( 7800): 	at android.content.ContentResolver.openTypedAssetFile(ContentResolver.java:1888)
E/AndroidRuntime( 7800): 	at android.content.ContentResolver.lambda$loadThumbnail$0(ContentResolver.java:4040)
E/AndroidRuntime( 7800): 	at android.content.-$$Lambda$ContentResolver$7ILY1SWNxC2xhk-fQUG6tAXW9Ik.call(Unknown Source:10)
E/AndroidRuntime( 7800): 	at android.graphics.ImageDecoder$CallableSource.createImageDecoder(ImageDecoder.java:558)
E/AndroidRuntime( 7800): 	at android.graphics.ImageDecoder.decodeBitmapImpl(ImageDecoder.java:1862)
E/AndroidRuntime( 7800): 	at android.graphics.ImageDecoder.decodeBitmap(ImageDecoder.java:1855)
E/AndroidRuntime( 7800): 	at android.content.ContentResolver.loadThumbnail(ContentResolver.java:4039)
E/AndroidRuntime( 7800): 	at android.content.ContentResolver.loadThumbnail(ContentResolver.java:4023)
E/AndroidRuntime( 7800): 	at com.morbit.photogallery.PhotoGalleryPlugin.getImageThumbnail(PhotoGalleryPlugin.kt:405)
E/AndroidRuntime( 7800): 	at com.morbit.photogallery.PhotoGalleryPlugin.getThumbnail(PhotoGalleryPlugin.kt:394)
E/AndroidRuntime( 7800): 	at com.morbit.photogallery.PhotoGalleryPlugin.access$getThumbnail(PhotoGalleryPlugin.kt:22)
E/AndroidRuntime( 7800): 	at com.morbit.photogallery.PhotoGalleryPlugin$onMethodCall$7.invoke(PhotoGalleryPlugin.kt:118)
E/AndroidRuntime( 7800): 	at com.morbit.photogallery.PhotoGalleryPlugin$onMethodCall$7.invoke(PhotoGalleryPlugin.kt:22)
E/AndroidRuntime( 7800): 	at com.morbit.photogallery.BackgroundAsyncTask.doInBackground(PhotoGalleryPlugin.kt:619)
E/AndroidRuntime( 7800): 	at com.morbit.photogallery.BackgroundAsyncTask.doInBackground(PhotoGalleryPlugin.kt:613)
E/AndroidRuntime( 7800): 	at android.os.AsyncTask$3.call(AsyncTask.java:394)
E/AndroidRuntime( 7800): 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
E/AndroidRuntime( 7800): 	... 4 more
I/Process ( 7800): Sending signal. PID: 7800 SIG: 9
```

<img width="277" alt="image" src="https://user-images.githubusercontent.com/1456389/90613313-dab77900-e211-11ea-9ef2-ed61aa5eff4e.png">

**Comment by wildsurfer** [comment]
![ezgif-7-f4afb7d5ab72](https://user-images.githubusercontent.com/1456389/90622270-8d410900-e21d-11ea-8cf0-fb23ebb444d7.gif)

**Comment by Firelands128** [comment]
I have tested on Android API 28,29,30 for both Google APIs and Google Play images in Android emulators, but I cannot reproduce this crash. And I'm running on MacOS X 10.15.5.
You can try it on some other emulators and Android API versions to check if they are working.

**Comment by wildsurfer** [comment]
Looks like I found the roots of the issue :)

Somehow I had a zero-byte image without extension in it's name in my photos library. It was causing this behaviour. When I created a new Android emulator instance example code starts working perfectly.

But there is still a problem with exact same error message but it is related to videos in a gallery. Will report it in next ticket. 

<img width="410" alt="Screenshot 2020-08-21 at 16 35 12" src="https://user-images.githubusercontent.com/1456389/90902109-a4762700-e3d4-11ea-8fa3-70b1885eb2b5.png">
<img width="410" alt="Screenshot 2020-08-21 at 16 34 58" src="https://user-images.githubusercontent.com/1456389/90902118-a8a24480-e3d4-11ea-81a1-1cab23d83fbc.png">
<img width="410" alt="Screenshot 2020-08-21 at 16 34 48" src="https://user-images.githubusercontent.com/1456389/90902122-a9d37180-e3d4-11ea-87e5-089fa143d20c.png">
